example/index.html

```diff
@@ -73,7 +73,13 @@
 		label {
 			position: relative;
 			overflow: hidden;
-			outline-style: dashed !important;
+		}
+
+		.file-upload-label.focus {
+			outline-offset: 8px;
```

Comment:
Should this not be 0 to test the difference?

Reply:
Yeah you're probably right... I was just a little put off by how that looks on this particular element:
![image](https://user-images.githubusercontent.com/32796923/99647608-4f3cc680-2a52-11eb-955b-ac84561e84fc.png)

Reply:
Maybe on a different element then?